lib/events/export/date_exporter.go

```diff
@@ -84,6 +95,14 @@ func (cfg *DateExporterConfig) CheckAndSetDefaults() error {
 	if cfg.PollInterval == 0 {
 		cfg.PollInterval = 16 * time.Second
 	}
+	if cfg.BatchExport != nil {
+		if cfg.BatchExport.MaxDelay == 0 {
+			cfg.BatchExport.MaxDelay = time.Second * 5
+		}
+		if cfg.BatchExport.MaxSize == 0 {
+			cfg.BatchExport.MaxSize = 2 * 1024 * 1024 // 2MiB
+		}
```

Comment:
```suggestion
		cfg.BatchExport.MaxDelay = cmp.Or(cfg.BatchExport.MaxDelay, 5 * time.Second)
		cfg.BatchExport.MaxSize = cmp.Or(cfg.BatchExport.MaxSize, 2 * 1024 * 1024 /* 2MiB */)
```

Reply:
done, I've updated the other fields to use `cmp.Or` for defaults as well, for consistency. Same in exporter.go.

Reply:
Isn't `cmp.Or` just the best?